src/components/Footer.astro

```diff
@@ -36,12 +36,12 @@ const groups: LinkGroup[] = [
 	{
 		title: 'Ecosystem',
 		links: [
-			{ label: 'Studio', href: 'https://studio.astro.build/' },
 			{ label: 'Community', href: 'https://community.astro.build' },
 			{
 				label: 'Contributing',
 				href: 'https://github.com/withastro/astro/blob/main/CONTRIBUTING.md',
 			},
+			{ label: 'Sponsors', href: 'https://opencollective.com/astrodotbuild#category-CONTRIBUTE' },
```

Comment:
Should this not include the link to a sub section?
```suggestion
			{ label: 'Sponsors', href: 'https://opencollective.com/astrodotbuild' },
```

Reply:
My thinking was that taking it to contribute is the call to action, plus that's where our sponsors themselves are most visible, but comparing that to the main link it basically gets you right there anyway. Sound good!